Sources/DropdownController.swift

```diff
@@ -21,9 +21,16 @@ open class DropdownController: UIViewController {
   public convenience init?(contentController: UIViewController, navigationController: UINavigationController) {
     guard let containerView = navigationController.tabBarController?.view ?? navigationController.view
       else { return nil }
-
-    let offsetY = navigationController.navigationBar.frame.maxY
-
+    var offsetY: CGFloat = 0
```

Comment:
I think you can just `let offsetY: CGFloat` because all the `if else` guarantee that it will get a value

Reply:
I agree with you, I think it's even better